package.json

```diff
@@ -7,12 +8,12 @@
   "license": "MPL-2.0",
   "preview": false,
   "private": true,
-  "engines": {
-    "vscode": "^1.41.0"
-  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/hashicorp/vscode-sentinel.git"
+    "url": "https://github.com/glenngillen/vscode-sentinel.git"
```

Comment:
was this intentional?

Reply:
Opps, bad rebase. Nothing to see here. 🎶